main.go

```diff
@@ -116,18 +119,25 @@ func createTraceFromJob(cfg *libhoney.Config, j Job) (*libhoney.Event, error) {
 	ev := createEvent(cfg)
 	defer ev.Send()
 	ev.Add(map[string]interface{}{
-		"ci_provider":     "GitLab-CI",
+		// Basic trace information
 		"service_name":    "job",
 		"trace.span_id":   spanID,
 		"trace.trace_id":  parentTraceID,
 		"trace.parent_id": parentTraceID,
 		"name":            fmt.Sprintf(j.BuildName),
-		"branch":          j.Ref,
-		"build_num":       j.PipelineID,
-		"build_id":        j.BuildID,
-		"repo":            j.Repository.Homepage,
+
+		// CI information
+		"ci_provider": "GitLab-CI",
+		"branch":      j.Ref,
+		"build_num":   j.PipelineID,
+		"build_id":    j.BuildID,
+		"repo":        j.Repository.Homepage,
 		// TODO: Something with job status
 		"status": j.BuildStatus,
+
+		// Runner information
+		"ci_runner":    j.Runner.Description,
+		"ci_runner_id": j.Runner.ID,
```

Comment:
I can imagine runner's `tags` would useful if some setup make proper use of them: i.e. tagging runners by CPU architecture to test a build on multi-platforms. 

In such cases, including `tags` might be a good way to correlate the events that only affect a certain type of CPU architecture but not others.

Reply:
My concern was it being an array and not a string, but you're right, I should stop being lazy and just `strings.Join` them or something, since then you can do `contains` queries.